src/Message.cpp

```diff
@@ -771,6 +780,10 @@ MethodReply MethodCall::sendWithReply(uint64_t timeout) const
 
     SDBUS_THROW_ERROR_IF(r < 0, "Failed to call method", -r);
 
+    if (connection_) {
+        connection_->notifyEventLoopNewTimeout();
+    }
```

Comment:
@riuriuriu @BenKaufmann Sorry to be late to the party, but do we need to send timeout notification here? It's always a synchronous call...